content/docs/command-reference/add.md

```diff
@@ -344,3 +331,50 @@ $ tree .dvc/cache
 
 Only the hash values of the `dir/` directory (with `.dir` file extension) and
 `file2` have been cached.
+
+## Example: Transfer to remote storage
+
+When you have a large dataset in an external location, you may want to track it
+as if it was in your project, but without downloading it locally (for now). The
+`--to-remote` option lets you do so, while storing a copy
+[remotely](/doc/command-reference/remote) so it can be
+[pulled](/doc/command-reference/plots) later. Let's initialize a DVC project,
```

Comment:
We're still explaining this almost exactly as in https://dvc.org/doc/command-reference/import-url#example-transfer-to-remote-storage but they're supposed to be completely different use cases. Is "you may want to track it as if it was in your project" clear and different enough?

How should we best differentiate them based on the discussions in #5445? Cc @shcheklein

Reply:
Perhaps we can defer this discussion. I am awaiting this PR to be merged so that I can work on to-cache docs.

Reply:
Yeah, I would write a better intro in general here:

By default when you do `import-url` DVC downloads data into the workspace so that it can be saved into cache, and later into the remote storage. That's important to preserve it since we want to keep the project reproducible. In some situations though you might not have enough space on the machine you are running `import-url`, but you still want this data to be saved into remote storage, you still want this data be accessible through regular commands like `dvc pull` (e.g. to run the pipeline on another machine that has enough space in cache, or when a large shared cache is being used, etc). In those cases, to "bootstrap" the project it's handy to use `--to-remote` ....

@jorgeorpinel we can take it over and rewrite it a bit, but let's not block @isidentical , and if we do this let's try to do this asap please or even as a separate PR

Reply:
> defer this discussion. I am awaiting this PR to be merged
> let's not block @isidentical

I approved the PR along with my comment.

Reply:
> let's try to do this asap please or even as a separate PR

Is it possible to review #2182? I have lots of other stashed changes on that branch and this will get added to that waitlist for now. Thanks

Reply:
@jorgeorpinel I guess you din't request a review from anyone for the #2182 :)

Reply:
p.s. Also added checkbox to #2121 for now (will come back to that one)

Reply:
> din't request a review

True! Usually they get reviewed without me asking haha but now that the team is growing I should be more explicit. Also per https://github.com/iterative/dvc.org/pull/2154#issuecomment-778440192 I've been avoiding pushing non-2.0 docs.

Reply:
p.s. guys I finally got to rewriting these examples a bit in #2302. Feel free to check it out.